### PR TITLE
Add support for span to RuleHelpXamlTranslator 

### LINKED ISSUE #4202: Add support for span to RuleHelpXamlTranslator

### PR DESCRIPTION
**Pull request by ugras-ergun-sonarsource**
fixes #4202

### CODE REVIEW

File: src/Education/XamlGenerator/RuleHelpXamlTranslator.cs [hunk 1]
```diff
@@ -307,6 +309,11 @@ private void ProcessElement()
 
                         break;
 
+                    case "span":
+                        WriteInlineElementStart("Span");
```
Comment (georgii-borovinskikh-sonarsource) [inline src/Education/XamlGenerator/RuleHelpXamlTranslator.cs:313]: Does it display correctly? Also add a simple test for this, maybe even parse to make sure

**Review by georgii-borovinskikh-sonarsource — CHANGES_REQUESTED** [review]
I think a test case is needed here

File: src/Education/XamlGenerator/RuleHelpXamlTranslator.cs (hunk 1 above)
Comment (ugras-ergun-sonarsource) [reply src/Education/XamlGenerator/RuleHelpXamlTranslator.cs:313]: Tests are in place already, that's how we have found the issue. And I have manually checked for failing rules and they were displayed correctly.

Comment (georgii-borovinskikh-sonarsource) [reply src/Education/XamlGenerator/RuleHelpXamlTranslator.cs:313]: Those are smoke tests. We also have `RuleHelpXamlTranslatorTests`. It's not a complete set of test cases, but IMO it's good to add tests for new stuff we add, especially if it's small things

**Review by georgii-borovinskikh-sonarsource — APPROVED** [review]

**Pull request merged**